### ISSUE TITLE
Add Browser selection support

### ISSUE THREAD

**Issue by piotrzarzycki21** [Moonshine-IDE/Super.Human.Installer#81]
We need to be able to pickup default browser which will be opened once server creation has been finished.

**Comment by piotrzarzycki21** [comment]
We have decided to have that support under **Settings**

**Comment by piotrzarzycki21** [comment]
I have pushed an update where:
- Open selected browser on Windows should work now
- we can now change default browser using list - without need of opening edit

New development build is available - it need to be installed manually -> https://github.com/Moonshine-IDE/Super.Human.Installer/releases/tag/0.8.18-dev

**Comment by JoelProminic** [comment]
Some cosmetic issues from today's discussion:

The error message has bad contrast here.
![image](https://github.com/Moonshine-IDE/Super.Human.Installer/assets/24532731/106c95ff-da9f-4cff-acc8-a7d1174a8621)

The user should not be able to make an unconfigured browser the default:
![image](https://github.com/Moonshine-IDE/Super.Human.Installer/assets/24532731/f714f493-ef9e-4a65-80b7-f42f3987dd9b)

**Comment by piotrzarzycki21** [comment]
@JoelProminic I have changed interface to have browsers settings in setting page - It's less confusing for the user.
- All links in application now opens in selected browser including welcome page once server finish start
Newest build is here: https://github.com/Moonshine-IDE/Super.Human.Installer/releases/tag/0.8.18-dev

I would like to release that version after our tomorrow's meeting.

**Comment by JoelProminic** [comment]
The update looks good.  I think moving the Browser list to be directly in Settings makes sense:
![image](https://github.com/Moonshine-IDE/Super.Human.Installer/assets/24532731/a9ab4780-9fbc-4688-9166-f375a9c6da9a)

One minor issue I noticed when testing links is that the Brave download link actually opens the Firefox download page.  We should do a similar test on Windows before the release

**Comment by JoelProminic** [comment]
I did a quick test on Windows and confirmed that the download URLs looked correct.  I was testing on a Vagrant VM, so I wasn't able to test the URL trigger after the VM was created.

Other known remaining issues:
- Need to clarify that Firefox is the recommended browser.  
- Can't edit browser paths.  I think @piotrzarzycki21 was planning this for the next release.

**Comment by piotrzarzycki21** [comment]
Thanks for testing! I see that you still doesn't have Opera on this list. I'm not sure why it's happening at your machine. I remember we were cleaning your browser part of configuration and still it did not help. I will think about it tomorrow morning more.

**Comment by JoelProminic** [comment]
Ah, I did wonder about that when I saw Opera on Windows.

On the note of Opera, I did some tests with installing each of the browsers, and I found that the Windows path for Opera did not work for me:
![image](https://github.com/Moonshine-IDE/Super.Human.Installer/assets/24532731/37b80e46-acb9-42cc-8c49-0842be7279f5)

The paths seem to match up, except for `${userName}`.  It seems this insertion parameter didn't work.

I checked for a Program Files entry for Opera, but I don't see it either.

On a side note, this reminded me of the contrast problems for the "%browser% not detected" messages.  My setup is making the problem worse, but I think the red text needs to be a little brighter, or it needs a (dark gray?) background.
